data-es2016plus.js

```diff
@@ -368,6 +368,101 @@ exports.tests = [
     ],
   },
   {
+    name: 'import()',
+    categpry: '2017 features',
+    significance: 'small',
+    spec: 'https://tc39.github.io/proposal-dynamic-import/',
+    subtests: [
+      {
+        name: 'import',
+        exec: function () {/*
+          try {
+            return typeof import === "function";
+            return true;
+          } catch {
+            return false;
+          }
+        */},
+        res: {
+        }
+      },
+      {
+        name: 'return',
+        exec: function () {/*
+          return import("./foo.js") instanceof Promise;
+        */},
+        res: {
+        }
+      },
+      {
+        name: 'throw',
+        exec: function () {/*
+          try {
+            import("")
```

Comment:
wouldn't `import('')` potentially reject anyways, since that's not a valid path?